src/hotspot/share/gc/serial/serialHeap.cpp

```diff
@@ -982,11 +904,7 @@ bool SerialHeap::print_location(outputStream* st, void* addr) const {
 }
 
 void SerialHeap::print_tracing_info() const {
-  if (log_is_enabled(Debug, gc, heap, exit)) {
-    LogStreamHandle(Debug, gc, heap, exit) lsh;
-    _young_gen->print_summary_info_on(&lsh);
-    _old_gen->print_summary_info_on(&lsh);
-  }
+ // Nothing
```

Comment:
What is the `Nothing` supposed to convey here?

Reply:
To emphasize that this empty method is intentional, inspired by `ZCollectedHeap::print_tracing_info`.

Reply:
Then better to keep the verb in the comment ` // Does nothing`